docs/content/FormControl.mdx

```diff
@@ -0,0 +1,414 @@
+---
+componentId: formControl
+title: FormControl
+status: Alpha
+description: The FormControl component is used to render a labelled text input and, optionally, associated validation text and hint text.
+source: https://github.com/primer/react/blob/main/src/FormControl/FormControl.tsx
+storybook: '/react/storybook?path=/story/forms-inputfield--text-input-field'
+---
+
+import {FormControl, TextInputWithTokens, Autocomplete, Select, Textarea, Checkbox, Radio} from '@primer/react'
+import {MarkGithubIcon} from '@primer/octicons-react'
+
+## Examples
+
+### Basic
+
+```jsx live
+<FormControl>
+  <FormControl.Label>Name</FormControl.Label>
+  <TextInput />
+</FormControl>
+```
+
+### `variant="stack"` (default) inputs
+
+```javascript live noinline
+const DifferentInputs = () => {
+  const [tokens, setTokens] = React.useState([
+    {text: 'zero', id: 0},
+    {text: 'one', id: 1},
+    {text: 'two', id: 2}
+  ])
+  const onTokenRemove = tokenId => {
+    setTokens(tokens.filter(token => token.id !== tokenId))
+  }
+
+  return (
+    <Box
+      display="flex"
+      flexDirection="column"
+      sx={{
+        '> * + *': {
+          marginTop: 4
+        }
+      }}
+    >
```

Comment:
I think this will give you a similar result with less code:
```suggestion
    <Box display="grid" gridGap={1}>
```

`<Box sx={{display: 'grid', gap: 1}}>` should also work, depending on your preference.